src/main/java/dude/Ui.java

```diff
@@ -0,0 +1,67 @@
+package dude;
+
+// deals with interactions with the user
+import dude.task.Task;
+
+import java.util.Scanner;
+public class Ui {
+    Scanner sc;
```

Comment:
Logical units within a block should be separated by one blank line.
Consider making the variable 'sc' private as well!